requirements.txt

```diff
@@ -1,4 +1,8 @@
 Django==2.2
 djangorestframework==3.9.2
+GDAL==2.4.1
+nose==1.3.7
+numpy==1.16.2
```

Comment:
Please update the requirements file. Packages like numpy and GDAL might not be used in our Django project

Reply:
I have been able to apply migrations and run the app. Just add psychopg2_binary to the requirements file.

Reply:
@kalsmic yeah am also wondering how they got there

@bisonlou  thats the same thing checkout . https://pypi.org/project/psycopg2-binary/

Reply:
@nadralia, they might installed system-wide on your computer, this way when you create the virtual environment, system-wide packages come along as well otherwise I do not know how else.